src/server/pkg/storage/chunk/storage.go

```diff
@@ -0,0 +1,50 @@
+package chunk
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"io/ioutil"
+
+	"github.com/pachyderm/pachyderm/src/server/pkg/obj"
+)
+
+// Storage is the abstraction that manages chunk storage.
+type Storage struct {
+	objC   obj.Client
+	prefix string
+}
```

Comment:
The name `Storage` feels too generic to me here, especially in a package called `chunk` could we just call this `Chunk` or `Chunker` or maybe `ChunkStorage`?

Reply:
`ChunkStorage` would be accomplished when you create this type through `chunk.Storage`. That is the Go way.

Reply:
That's a fair point... I think of all my suggestions only `Chunker` would pass the linter, so yeah, forget I said anything, `chunk.Storage` is a good idiomatic go name.